libecl/src/ecl_grid.c

```diff
@@ -4002,28 +4003,29 @@ static bool concave_cell_contains( const ecl_cell_type * cell, int method, const
 
   // Test for containment in cell core
   bool contained = tetrahedron_by_points_contains(dia[0][0], dia[1][0], dia[0][1], dia[1][1], p);
+  bool core_sign = tetrahedron_positive_volume(dia[0][0], dia[1][0], dia[0][1], dia[1][1]);
```

Comment:
I guess the question here is left-handedness versus right handedness; i.e. the variable `core_sign` could be renamed:
```
bool right_handed = tetrahedron_positive_volume( ... )
```